src/test/java/com/google/api/codegen/testdata/python_library.baseline

```diff
@@ -919,15 +919,15 @@ class Timestamp(object):
     0001-01-01T00:00:00Z to 9999-12-31T23:59:59.999999999Z.
     By restricting to that range, we ensure that we can convert to
     and from  RFC 3339 date strings.
-    See [https://www.ietf.org/rfc/rfc3339.txt](https://www.ietf.org/rfc/rfc3339.txt).
+    See `https://www.ietf.org/rfc/rfc3339.txt <https://cloud.google.comhttps://www.ietf.org/rfc/rfc3339.txt>`_.
```

Comment:
Yes, this is wrong. IIRC PythonSphinxCommentFixer adds this cloud.google.com somehow, you need to fix there (and the same problem exists in RDocCommentFixer)